contrib/filter-repo-demos/lint-history

```diff
@@ -160,6 +172,10 @@ if lint_args.relevant:
   exec('def is_relevant(filename):\n  '+'\n  '.join(body.splitlines()),
        globals())
   lint_args.filenames_important = True
+if lint_args.callback:
+  body= lint_args.callback
```

Comment:
Why a space after the `=` but not before?  I've seen some folks that like no spaces, and some that like spaces on each side, but never this.  Regardless, style in the rest of the file is a space on each side.